cmd/emp/main.go

```diff
@@ -103,6 +109,34 @@ func (c *Command) ShortExtra() string {
 	return c.Short[:len(c.Short)-len(extra)]
 }
 
+type Output struct{}
+
+func (t *Output) JSONOutput(o interface{}) error {
+	b, err := json.Marshal(o)
+	if err == nil {
+		fmt.Println(string(b))
+	}
+	return err
+}
+
+type CommandOutput interface {
```

Comment:
I would probably make these take an io.Writer as the first argument, which they should write their output to. Also makes it easier to test the outputs, if we ever need to:

```go
type CommandOutput interface {
        WriteJSON(io.Writer) error
        WriteHuman(io.Writer) error
}
```

Or something like that.

Could also use interface upgrades to check if an output supports a certain format:

```go
func (c *Command) WriteOutput(v interface{}) error {
        w := os.Stdout

        switch ot {
        case "json":
                return json.NewEncoder(w).Encode(v)
        case "human":
                if w, ok := v.(HumanOutput); ok {
                        return w.WriteHuman(w)
                }
        }
}
```